### PR TITLE
{data,math}[intel/2019a] GEOS v3.7.2, GDAL v3.0.0 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node3122.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c5df0a1237d3228ebe59de53dc74f8cd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2093.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/92a527b19fea7918f9fd41b7ed8a65c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2613.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/50c14a88a998f7ed4e3d9c3817fba46e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3122.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/63ff46235ef3698da63656d513dc2c86 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2093.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3e036df7b2176618fcaa1421e194e864 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2613.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a624e42d8082be0d7a25390dbdc32916 for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-intel-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://trac.osgeo.org/geos'
```
Comment (Micket) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-intel-2019a-Python-3.7.2.eb]: https

File: easybuild/easyconfigs/g/GDAL/GDAL-3.0.0-intel-2019a-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '3.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.gdal.org/'
```
Comment (Micket) [inline easybuild/easyconfigs/g/GDAL/GDAL-3.0.0-intel-2019a-Python-3.7.2.eb]: https

File: easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-intel-2019a-Python-3.7.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-intel-2019a-Python-3.7.2.eb]: fixed in 1be1b15273

File: easybuild/easyconfigs/g/GDAL/GDAL-3.0.0-intel-2019a-Python-3.7.2.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GDAL/GDAL-3.0.0-intel-2019a-Python-3.7.2.eb]: fixed in 1be1b15273

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/a11c974ab83981f198b801692f88385e for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**